Makefile

```diff
@@ -38,7 +38,8 @@ OTELCOL_BINARY=google-cloudops-opentelemetry-collector
 
 .PHONY: build
 build:
-	go build -o ./bin/$(OTELCOL_BINARY)_$(GOOS)_$(GOARCH)$(EXTENSION) ./cmd/otelopscol
+	@echo "Building collector binary..."
```

Comment:
Do we need these? Doesn't make print out the command it's running anyway?

Reply:
Yes it does, but I find them not so readable, let me know if you find these unnecessary.

Reply:
I dunno, it feels a bit overkill imo. The target name already describes what it does

Reply:
That's true. Since it is mainly engineers who will be using this, I can see how these outputs don't necessarily help with this group of users. I just removed these.